src/test/java/io/crate/client/jdbc/integrationtests/CrateJDBCIntegrationTest.java

```diff
@@ -182,12 +207,14 @@ public void setUpTable() {
     @After
     public void tearDownTable() {
         CrateClient client = new CrateClient(hostAndPort);
-        try {
-            client.sql("drop table test").actionGet();
-            client.sql("drop table if exists foo.t").actionGet();
-            client.sql("drop table if exists bar.t").actionGet();
-        } catch (Exception e) {
-            // ignore
+        SQLResponse response = client.sql("select schema_name, table_name from information_schema.tables where schema_name in ('doc', 's1', 's2', 'foo', 'bar')").actionGet();
+        for (Object[] row : response.rows()) {
+            try {
+                client.sql(String.format("drop table \"%s\".\"%s\"", row[0], row[1]));
+                Thread.sleep(500);
```

Comment:
@bogensberger problem is re-creating the same table immediately.

probably need to change the test setup then, using different table names each time